### PR TITLE
IOW-587: Site_Active added to Time Series Services

### PR DESCRIPTION
**Pull request by dsiemon2**
IOW-587: Site_Active added to Time Series Services
Tests added

### CODE REVIEW

File: src/main/resources/mybatis/features.xml [hunk 1]
```diff
@@ -39,6 +39,14 @@
         <if test="monitoringLocationType != null">
             and site_type in <foreach item="i" collection="monitoringLocationType" open="(" separator="," close=")">#{i}</foreach>
         </if>
+        <if test="siteActive != null">
+            <if test="siteActive ==  TRUE">
```
Comment (kkehl-usgs) [inline src/main/resources/mybatis/features.xml:43]: extra whitespace after the equals sign?

File: src/main/java/gov/usgs/wma/waterdata/collections/CollectionsController.java [hunk 2]
```diff
@@ -184,6 +184,10 @@ public String getOgcCollectionFeatures(
 		@Parameter(description="Well, Stream, or other type")
 		@RequestParam(value="monitoringLocationType", required = false) List<String> monitoringLocationType,
 
+		//@Size(min=1, max=100, message="The Active Flag for Site monitoring locations ")
```
Comment (kkehl-usgs) [inline src/main/java/gov/usgs/wma/waterdata/collections/CollectionsController.java:187]: can we remove this?

File: src/test/resources/testResult/featureCollection/monitoring-locations/featureCollection_s19l10.json [hunk 3]
```diff
@@ -38,6 +39,7 @@
         "countryFIPS": null,
         "hydrologicUnit": null,
         "monitoringLocationUrl": null
+
```
Comment (kkehl-usgs) [inline src/test/resources/testResult/featureCollection/monitoring-locations/featureCollection_s19l10.json:42]: extra line

**Review by kkehl-usgs — APPROVED** [review]

File: pom.xml [hunk 4]
```diff
@@ -199,7 +199,7 @@
                             <goal>stop</goal>
                         </goals>
                     </execution>
-                </executions>
+                </executions>
```
Comment (orosz-usgs) [inline pom.xml:202]: extra space

File: src/test/java/gov/usgs/wma/waterdata/collections/FeaturesFilterIT.java [hunk 5]
```diff
@@ -70,6 +70,26 @@ public void stateTestNotFound() throws IOException {
...
+            String compareFile = "featuresFilter/monitoring-locations/monitoring_location_type_multi_values.json";
+            doJsonCompare(rtn, compareFile);
+    }
+
+    @Test
+    public void siteActiveTestFalseFound() throws IOException {
+        ResponseEntity<String> rtn = restTemplate.getForEntity("/collections/monitoring-locations/items?siteActive=false",
+            String.class);
+            assertThat(rtn.getStatusCode(), equalTo(HttpStatus.OK));
+            String compareFile = "featuresFilter/monitoring-locations/monitoring_location_type_multi_values.json";
```
Comment (orosz-usgs) [inline src/test/java/gov/usgs/wma/waterdata/collections/FeaturesFilterIT.java:87]: The compare file is the same for both tests.

File: src/test/resources/testResult/featureCollection/monitoring-locations/featureCollection.json [hunk 6]
```diff
@@ -175,6 +176,7 @@
                 "state": null,
                 "stateFIPS": null,
                 "wellDepth": null
+
```
Comment (orosz-usgs) [inline src/test/resources/testResult/featureCollection/monitoring-locations/featureCollection.json:179]: extra line

**Review by orosz-usgs — CHANGES_REQUESTED** [review]

File: src/test/resources/testResult/featuresFilter/monitoring-locations/monitoring_location_site_active_false.json [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+      "type": "text/html",
+      "rel": "canonical",
+      "title": "information",
+      "href": "https://waterdata.usgs.gov"
+    },
+    {
+      "type": "application/geo+json",
+      "rel": "self",
+      "title": "This document as GeoJSON",
+      "href": "https://unittest/api/observations/collections/monitoring-locations/items?f=json&startIndex=0&limit=10000"
```
Comment (orosz-usgs) [inline src/test/resources/testResult/featuresFilter/monitoring-locations/monitoring_location_site_active_false.json:70]: This url should have '&siteActive=false'

File: src/test/resources/testResult/featuresFilter/monitoring-locations/monitoring_location_site_active_true.json [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+      "type": "text/html",
+      "rel": "canonical",
+      "title": "information",
+      "href": "https://waterdata.usgs.gov"
+    },
+    {
+      "type": "application/geo+json",
+      "rel": "self",
+      "title": "This document as GeoJSON",
+      "href": "https://unittest/api/observations/collections/monitoring-locations/items?f=json&startIndex=0&limit=10000"
```
Comment (orosz-usgs) [inline src/test/resources/testResult/featuresFilter/monitoring-locations/monitoring_location_site_active_true.json:70]: This url should have '&siteActive=true'

**Review by orosz-usgs — CHANGES_REQUESTED** [review]

**Pull request merged**